src/arch/helperrvv.h

```diff
@@ -548,6 +603,17 @@ static INLINE vmask vand_vm_vo32_vm(vopmask x, vmask y) {
 static INLINE vmask vandnot_vm_vo32_vm(vopmask x, vmask y) {
   return __riscv_vmerge(y, 0, x, VECTLENSP);
 }
+#if defined(ENABLE_RVVM1) || defined(ENABLE_RVVM1NOFMA)
```

Comment:
If you can't use macros given the operation is different between RVVM1 and RVVM2, please put the #ifdef _inside_ the function to at least keep the signature in common.